src/hl/ready.rs

```diff
@@ -396,7 +404,11 @@ where
     /// and more. Make sure that the values used are the same as of the frames
     /// that are transmitted. The default works with the TxConfig's default and
     /// is a sane starting point.
-    pub fn receive(self, config: Config) -> Result<DW3000<SPI, SingleBufferReceiving>, Error<SPI>> {
+    pub fn receive(
```

Comment:
Please add a `receive` wrapper that have the same signature as before, and rename this to be `receive_delayed`